worlds/tloz/Rules.py

```diff
@@ -0,0 +1,82 @@
+import typing
```

Comment:
this entire file appears to be a copy-paste from the generic world, please just import that instead.